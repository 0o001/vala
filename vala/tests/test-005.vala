namespace Maman {
	class Bar {
		public static int do_action (int i) {
			return (i + 42);
		}
	}
	
	class SubBar : Bar {
		public static int main (int argc, string[] argv) {
			int i = do_action (42);
			return (argc + i);
		}
	}
}
