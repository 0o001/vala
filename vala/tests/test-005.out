Static Inheritance Test: 1 2 3
